### ISSUE TITLE
Windows overview mode should be keyboard navigable like Unity

### ISSUE THREAD

**Issue by uncertainquark** [elementary/gala#219]
Unity can navigate between windows in the overview mode using keyboard arrow keys which is great.
![screenshot from 2018-03-30 13-14-49](https://user-images.githubusercontent.com/22585732/38159251-389ea656-34c2-11e8-8845-4b17df1faa1e.png)

It also aids when you search and filter windows, as suggested in #218.

**Comment by cassidyjames** [comment]
The window overview and workspace overview are pretty redundant, and honestly I think we want to merge them. 

>There is a similar function on the workspace overview, but is unpredictable.

Then we should report and fix issues against keyboard navigation in the workspace overview. :stuck_out_tongue:

**Comment by Astro-City** [comment]
> The window overview and workspace overview are pretty redundant, and honestly I think we want to merge them.

That would be great, but the Super+A function would be lost.

**Comment by cassidyjames** [comment]
Since #287 is the way we want to move forward, I'm closing this as superseded.
